kobo/version.go

```diff
@@ -54,3 +55,16 @@ func ParseKoboAffiliate(kpath string) (affiliate string, err error) {
 	}
 	return m[1], nil
 }
+
+// ParseKoboUAString parses a web browser UA string for Kobo ID and version info
+func ParseKoboUAString(ua string) (version, id string, err error) {
+	m := regexp.MustCompile(`.+\(Kobo Touch (\d+)/(\d+\.\d+.\d+)\)`).FindStringSubmatch(ua)
+	if len(m) != 3 {
+		return "", "", errors.New("could not parse UA string")
+	}
+	idInt, err := strconv.ParseInt(strings.TrimLeft(m[1], "0"), 10, 32)
```

Comment:
```suggestion
	idInt, err := strconv.Atoi(m[1])
```

It isn't necessary to trim leading zeroes when parsing it as an integer, and `strconv.ParseInt` can be simplified to `strconv.Atoi`.

Reply:
Ah... (rereads the docs), leading zeros are only an issue if the base argument is 0.

Oops. Thanks for double checking that.